docs/user-guide/cli-uninstall.md

```diff
@@ -1,50 +1,86 @@
 
-**Important\!** The uninstall process does not delete the profiles and credentials that you created when using the product from your computer. To delete the profiles from your computer, delete them before you uninstall Zowe CLI.
+:::info Required roles: Systems administrator
+:::
 
-The following steps describe how to list the profiles that you created, delete the profiles, and uninstall Zowe CLI.
+:::info Important
 
-1. Open a command-line window.
+The uninstall process does not delete the profiles and credentials that you created when using the product from your computer. To delete the profiles from your computer, delete them before you uninstall Zowe CLI.
 
-    **Note:** If you do not want to delete the Zowe CLI profiles from your computer, go to Step 5.
+:::
+
+To list the profiles that you created, delete the profiles, and uninstall Zowe CLI.
+
+1. Open a command line window.
+
+    :::note
+    
+    If you do not want to delete the Zowe CLI profiles from your computer, proceed to Step 5.
+
+    :::
 
 2. List all configuration files that you created. Issue the following command:
 
     ```
     zowe config list --locations --root
     ```
-    **Example:**
+    Example response for Windows:
 
     ```
     $ zowe config list --locations --root
     C:\Users\SMITH-123\.zowe\zowe.config.json
     $
     ```
 
-3. Delete all of the configuration files that are listed. Issue the following command:
-
-    **Tip:** For this command, use the results of the `zowe config list`
-    command.
+    Example response for Linux and Mac OS:
 
     ```
-    rm C:\Users\SMITH-123\.zowe\zowe.config.json
+    $ zowe config list --locations --root
+    ~/.zowe/zowe.config.json
+    $
     ```
 
-    <!-- **Note:** When you issue the `delete` command, it deletes the
-    specified profile and its credentials from the credential vault in your computer's operating system. -->
+    
+
+3. Delete all of the configuration files that are listed. Issue the following command:
+
+    :::tip
+    
+    For this command, use the results of the `zowe config list` command.
+
+    :::
+
+    For Windows:
+
+        ```
+        del C:\Users\SMITH-123\.zowe\zowe.config.json
+        ```
+    For Linux and Mac OS:
+
+        ```
+        rm ~/.zowe/zowe.config.json
+        ```
 
 4. Uninstall Zowe CLI by issuing the following command:
 
     ```
     npm uninstall --global @zowe/cli
     ```
 
-    **Note:** You might receive an `ENOENT` error when issuing this command if you installed Zowe CLI from a local package (.tgz) and the package was moved from its original location. In the event that you receive the error, open an issue in the Zowe CLI GitHub repository.
+    :::note 
+
+    You might receive an `ENOENT` error when issuing this command if you installed Zowe CLI from a local package (.tgz) and the package was moved from its original location. In the event that you receive the error, open an issue in the Zowe CLI GitHub repository.
```

Comment:
This must've been a problem specific to a npm version that we no longer support, likely around the time that NPM had a lot of interesting breaking changes.

I did test (caveat: on docker containers) versions of node 14, 16, 18, 20, 21 (with their bundled npm versions, including npm7 on node 14-16) and the package manager (npm) was able to uninstall the CLI globally even though the original tgz was renamed, moved to different directories, and even deleted.

Worst case:
The user can just add the `--force` option to the `npm uninstall --global @zowe/cli` command and it will bypass any errors 😋 

Let's remove the suggestion to open a GitHub issue 😋

Reply:
@adam-wolfe @zFernand0 - would this note work better for us?

![image](https://github.com/zowe/docs-site/assets/109533923/466b6a71-cb3c-4ed8-8019-63ab4df0f066)